app/internal/event.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import re
+
+from email_validator import validate_email, EmailSyntaxError
```

Comment:
You can use `pydantic` instead :)

Reply:
Could, but why? Pydantic uses this library.